### PR TITLE
Combine worek work with bug fix

### PR DESCRIPTION
**Pull request by samiebee43**

### CODE REVIEW

File: pluginExamples/OEM/Worek.py [hunk 1]
```diff
@@ -48,35 +30,174 @@
...
+    ex: getY(4, [0,2,4,6,8], [1,3,5,7,9]) --> 5
+
+    if x is not in the domain, return the closest boundary element
+    """
+
+    for i in range(len(domain)-1):
+        if domain[i] <= x < domain[i+1]:
+            return codomain[i]
+
+    if domain[-1] >= x:
```
Comment (mzieg) [inline pluginExamples/OEM/Worek.py:57]: minor optimization note: you can check for endpoints (less than first, greater than last) before the loop.

File: pluginExamples/OEM/Worek.py [hunk 2]
```diff
@@ -48,35 +30,174 @@
...
+        for elem in self.enlighten_info.plugin_fields():
+            if elem.field_name == name:
+                widget = elem
+        return widget.field_widget
+
+    def graph_slope(self):
+        # self.get_widget_from_name("slope start") will be a QtWidgets.QDoubleSpinBox
+        # same thing for slope_end_txtbox
+
+        blank_start = self.get_widget_from_name("blank start").value()
```
Comment (mzieg) [inline pluginExamples/OEM/Worek.py:113]: Do the calls to the widgets mean that this function has to run from the foreground thread?

File: pluginExamples/OEM/Worek.py [hunk 3]
```diff
@@ -87,52 +208,82 @@ def connect(self, enlighten_info):
...
+
+            AchE = (Activity * 1.58 * 1000) / HbC
+
+            # these quantities are 
+            # - Sample rate (comes from slopes)
+            # - Blank rate
+            # --------------------------
+            # - Hemoglobin concentration (applying formulas)
+            # - Enzyme Activity (AChE, BChE)
+            # - Erythrocyte AChE
```
Comment (mzieg) [inline pluginExamples/OEM/Worek.py:268]: great docs

File: pluginExamples/OEM/Worek.py (hunk 2 above)
Comment (samiebee43) [reply pluginExamples/OEM/Worek.py:113]: No, this is happening from the worker thread.

File: pluginExamples/OEM/Worek.py (hunk 3 above)
Comment (samiebee43) [reply pluginExamples/OEM/Worek.py:268]: Ty

File: pluginExamples/OEM/Worek.py (hunk 2 above)
Comment (mzieg) [reply pluginExamples/OEM/Worek.py:113]: We're able to read Qt widgets from other threads?  Is it just write access that is problematic?

**Pull request merged**